.vscode/settings.json

```diff
@@ -19,6 +19,7 @@
   },
   "coverage-gutters.showGutterCoverage": false,
   "coverage-gutters.showLineCoverage": true,
+  "scss.lint.unknownAtRules": "ignore",
```

Comment:
Style: Alphabetical ordering should be inserted in above the `yaml.schemas` key.